packages/loaders/src/types/index.ts

```diff
@@ -14,7 +14,7 @@ export type Size = (typeof SIZE)[number];
 export interface IDotsProps extends SVGAttributes<SVGSVGElement> {
   /** Sets the height and width in pixels. Inherits the parent's font size by default. */
   size?: string | number;
-  /** Sets the fill color. Inherits the parent's `color` by default. */
+  /** Overrides the default fill color. */
```

Comment:
As mentioned ☝️ , breaking change?

Reply:
Again, I don't think this is something we should do.